core/src/main/java/greencity/controller/OrderController.java

```diff
@@ -408,7 +408,7 @@ public OrderController(UBSClientService ubsClientService) {
         PaymentResponseDtoLiqPay dto, HttpServletResponse response) throws IOException {
         ubsClientService.validateLiqPayPayment(dto);
         if (HttpStatus.OK.is2xxSuccessful()) {
-            response.sendRedirect("https://ita-social-projects.github.io/GreenCityClient/#/ubs/confirm");
+            response.sendRedirect("http://localhost:4200/#/ubs/confirm");
```

Comment:
why localhost? baseUrl should be taken from the properties. In each profile you set different value based on environment.